packages/prisma/src/data/seedTEST.ts

```diff
@@ -506,6 +506,62 @@ async function seedTest(prisma: Prisma.PrismaClient) {
       update: {},
     })
   })
+
+  const goldMedalAchievement = await prisma.achievement.upsert({
```

Comment:
run seedAchievements.ts in prisma:setup instead of duplicating the achievements code?

Reply:
done